packages/dapp/src/components/Main.tsx

```diff
@@ -39,22 +39,38 @@ import WrongNetwork from "./WrongNetwork";
 import config from "../helpers/config";
 import { State } from "../redux/reducers";
 import { supportedNetworks } from "../helpers/networkHelpers";
+import cookie from "react-cookies";
+import { addMiddleware } from "redux-dynamic-middlewares";
+import { gaMiddleware } from "../redux/analytics";
 
 export interface MainReduxProps {
   network: string;
 }
 
 export interface MainState {
   prevAccount: EthAddress;
+  cookieSet: boolean;
```

Comment:
Is this actually being used?

Reply:
kind of. it just modifies the state so we do a re-render and get rid of the AskTrackingBar

Reply:
but the value of it isn't used

Reply:
i guess `.forceUpdate()` is better, right?